packages/validation/src/validators/open-policy-agent/validator.ts

```diff
@@ -15,7 +15,7 @@ import { isDefined } from "../../utils/isDefined.js";
 import { OPEN_POLICY_AGENT_RULES } from "./rules.js";
 import { LoadedPolicy, OpaProperties, PolicyError } from "./types";
 import { WasmLoader } from "./wasmLoader/WasmLoader.js";
-import { isKustomizationResource } from "../../references/utils/kustomizeRefs";
+import { isKustomizationResource } from "../../references/utils/kustomizeRefs.js";
```

Comment:
Always be sure to end with `.js` or ESM builds will fail. I've disabled automated CLI releases until we can add sanity tests as this is easy to overlook.